src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -95,12 +101,18 @@ private Response performRequest(Request request) throws IOException {
     return Response.status(status).entity(body).build();
   }
 
-  public Response indexDatasetTerms(List<DatasetTerm> datasets) throws IOException {
+  public Response indexDatasetTerms(List<DatasetTerm> datasets, User user) throws IOException {
     List<String> bulkApiCall = new ArrayList<>();
 
     datasets.forEach(dsTerm -> {
-      bulkApiCall.add(bulkHeader.formatted(dsTerm.getDatasetId()));
+      bulkApiCall.add(BULK_HEADER.formatted(dsTerm.getDatasetId()));
       bulkApiCall.add(GsonUtil.getInstance().toJson(dsTerm) + "\n");
+      try {
+        updateDatasetIndex(dsTerm.getDatasetId(), user.getUserId(), Instant.now());
+      } catch (SQLException e) {
+        // We don't want to send these to Sentry, but we do want to log them for follow up off cycle
+        logWarn("Error updating dataset indexed date for dataset id: %d ".formatted(dsTerm.getDatasetId()), e);
```

Comment:
Should this logging and try/catch be done in `updateDatasetIndex()? There is one other call to this method where `null` is written, do we want that to also be logged?

Reply:
I can move the logging there, that would simplify this method a bit to avoid the try/catch